### PR TITLE
Adding download file and Merkle dir to remote tool.

### PR DESCRIPTION
**Pull request by ola-rozenfeld**

### CODE REVIEW

**Comment by ola-rozenfeld** [comment]
This also has a few minor fixes and cleanups: sorting the printed paths, always printing the full digests (including size), and not exposing any internal details in example command.

File: go/cmd/remotetool/main.go [hunk 1]
```diff
@@ -29,35 +36,28 @@ type OpType string
 const (
 	downloadActionResult OpType = "download_action_result"
 	showAction           OpType = "show_action"
+	downloadBlob         OpType = "download_blob"
+	downloadMerkleTree   OpType = "download_merkle_tree"
 )
 
 var (
-	operation  = flag.String("operation", "download_action_result", "Specifies the operation to perform. Currently only download_action_result is supported.")
+	operation  = flag.String("operation", "", "Specifies the operation to perform. Currently only download_action_result is supported.")
```
Comment (gkousik) [inline go/cmd/remotetool/main.go:44]: I think this should be removed and the appropriate options should be added.

File: go/cmd/remotetool/main.go [hunk 2]
```diff
@@ -29,35 +36,28 @@ type OpType string
 const (
 	downloadActionResult OpType = "download_action_result"
 	showAction           OpType = "show_action"
+	downloadBlob         OpType = "download_blob"
+	downloadMerkleTree   OpType = "download_merkle_tree"
```
Comment (gkousik) [inline go/cmd/remotetool/main.go:40]: probably name this download_tree / download_dir ?

File: go/pkg/tool/tool.go [hunk 3]
```diff
@@ -71,6 +77,55 @@ func (c *Client) DownloadActionResult(ctx context.Context, actionDigest, pathPre
...
+	}
+	dg, err := digest.NewFromString(blobDigest)
+	if err != nil {
+		return "", err
+	}
+	log.Infof("Downloading blob of %v to %v.", dg, path)
+	if _, err := c.GrpcClient.ReadBlobToFile(ctx, dg, path); err != nil {
+		return "", err
+	}
+	if outputToStdout {
```
Comment (gkousik) [inline go/pkg/tool/tool.go:105]: nit:
```
if !outputToStdout {
    return "", nil
}
contents, err := ioutil.ReadFile(path)
if err != nil {
    return "", err
}
return string(contents), nil
```

**Review by gkousik — APPROVED** [review]

File: go/cmd/remotetool/main.go (hunk 2 above)
Comment (ola-rozenfeld) [reply go/cmd/remotetool/main.go]: Done.

File: go/cmd/remotetool/main.go (hunk 1 above)
Comment (ola-rozenfeld) [reply go/cmd/remotetool/main.go]: Done.

File: go/pkg/tool/tool.go (hunk 3 above)
Comment (ola-rozenfeld) [reply go/pkg/tool/tool.go]: Done.

**Comment by ola-rozenfeld** [comment]
This addresses #89 as well.

**Pull request merged**